cli/dts/lib.deno.unstable.d.ts

```diff
@@ -480,7 +482,7 @@ declare namespace Deno {
    * The result of `Deno.emit()` API.
    */
   export interface EmitResult {
-    /** Diagnostic messages returned from the type checker (`tsc`). */
+    /** Diagnostic messages returned from the type checker (`tsc`). Can be used with `Deno.formatDiagnostics` to display a user friendly string. */
```

Comment:
Perhaps move the second sentence to a new paragraph? That way the top paragraph describes what it does at a glance and they can read the bottom paragraph for some extra information if they desire.

It seems we don't use `@remarks`, but it would make a good remark 😅

Reply:
Remarks aren't a js thing are they? Regardless i've actioned on the other part of your suggestion: https://github.com/denoland/deno/pull/10925/commits/29df9a3cc0f436281b96f526a156fca1a2f4a49c